src/odemis/gui/win/acquisition.py

```diff
@@ -923,8 +921,9 @@ def on_acquire(self, evt):
                 self._view.addStream(s)
 
         # TODO: Test with merged code from stitching
-        # self.acq_future, est_time, est_memory = acq.stitching.acquireOverviewMap(streams, self._main_data_model.stage)
-        self.acq_future = acqmng.acquire(streams, self._main_data_model.settings_obs)
+        self.acq_future, est_time, est_memory = stitching.acquireOverviewMap(streams, self._main_data_model.stage)
+        self.update_acquisition_time(est_time)
```

Comment:
Update it to the modified version `acquireTiledArea`. 
Need to pass area (left, top, right, bottom) parameter (Probably as stage metadata.)
Pass `self._main_data_model.settings_obs` to the function.

Reply:
@lazem , could you be more specific about how to get the area? Please, put the code needed to read the metadata from the stage and convert it to the right `area` argument.

Reply:
Sorry I just saw this now. Well I didn't really need to add the area in the metadata, but I included it in this commit so @amuskens can use it. 
https://github.com/delmic/odemis/pull/1455/commits/b016894200a90367990add95d3ecf5dc13831d08